_projects/2024/100535296/100535296.Rmd

```diff
@@ -0,0 +1,534 @@
+  ) + theme +
+  coord_fixed(ratio = 1.5)
+
+p
+
+```
+
+Next, we customized the graph's axes. For the X-axis, we used scale_x_continuous to position it at the top of the graph. This function also defines a range between 10 and 73, sets tick marks at intervals of 5, and customizes the labels to display "10YRS" only for the value 10, while showing the other values as standard numbers. For the Y-axis, we applied scale_y_discrete to remove unnecessary space before and after the categories (using expand = c(0, 0)), ensuring that the graph elements fit directly within the drawing area. After adjusting the axes, we added the title, subtitle, and caption (which will appear as the description in the top-right corner of the graph) using the labs function.
+
+```{r, fig.width=4, fig.height=6, out.width="100%"}
+p <- p + 
+  scale_x_continuous(
+    position = "top",
+    limits = c(10, 73),
+    breaks = seq(10, 70, by = 5),
+    labels = function(x) ifelse(x == 10, "10YRS", as.character(x))
+  ) +
+  scale_y_discrete(expand = c(0, 0)) +
+  labs(
+    title = "Olympians",
+    subtitle = "By age, sport, country, and sex",
+    y = "BY SPORT",
+    caption = "Explore a Washington Post breakdown of all Summer Olympics\n athletes who have competed since the first modern Games in 1896.\n See the expansive rang in age of the competitors, and when men\n and women first competed in each sport."
+    )  
+  
+
+p
+```
+In the next step, we used the geom_hline() function to add a white line to the graph, separating the sport categories. We position the line just below each category by converting the values of the Sport variable to numeric format and adjusting them by 0.5. 
+
+```{r, fig.width=4, fig.height=6, out.width="100%"}
+
+p <- p +
+  geom_hline(
+    aes(yintercept = as.numeric(Sport) - 0.5),
+    color = "white",
+    size = 0.3
+  )
+
+p
+```
+Next, we added the maximum and minimum age for each sport by using the variables max_age and min_age, which we created earlier. These values are then displayed on the graph using geom_text(), positioning the text labels accordingly.
+
+```{r, fig.width=4, fig.height=6, out.width="100%"}
+
+p <- p +geom_text(
+    aes(x = min_age, y = Sport, label = min_age),
+    size = 2.5,
+    hjust = 1.1,
+    color = "gray40",
+    family = "Arial"
+  ) +
+  geom_text(
+    aes(x = max_age, y = Sport, label = max_age),
+    size = 2.5,
+    hjust = -0.1,
+    color = "gray40",
+    family = "Arial")
+
+p
+```
+Finally, we add the annotations and the lines that connect the data points to these annotations. To create the L-shaped arrows, we combine two geom_segment() functions, one vertical and one horizontal, and position them on the plot to form this shape. Next, we add the three annotations visible in the graph using annotate(), specifying the exact location within the plot, as well as the size, color, font, and line spacing (with lineheight).
+
+```{r, fig.width=4, fig.height=6, out.width="100%"}
```

Comment:
Please add `preview = TRUE` so that this image is shown in the list of projects.